src/Evolution/Systems/Burgers/Observe.hpp

```diff
@@ -89,8 +93,14 @@ class Observe : public Event<EventRegistrars> {
 
   using options = tmpl::list<>;
   static constexpr OptionString help =
-      "Observes the Burgers field U and the error and error norm compared\n"
-      "to the analytic solution.";
+      "Observes the Burgers field U and its error.\n"
+      "\n"
+      "Writes volume quantities:\n"
+      " * U\n"
+      " * ErrorU = U - analytic solution\n"
```

Comment:
perhaps `U - U_analytic`? Not sure about the pseudo-math here, perhaps add some text that defines `U_analytic`? (It seems important to get this OptionString right because it will set an example for those to follow.)

As a guideline we could format this as:
`  * QuantityName = Expression: Description`

Reply:
I don't understand what you are suggesting, sorry.  Could you give an example or some suggested text?

Reply:
@nilsleiffischer ping

Reply:
If you just add the other observed quantities that I commented on below, this should be fine. I‘m just worried that other events where the quantities may be less obvious will not be sufficiently documented, but we’ll deal with that in later PRs.